.github/workflows/test-suite-desktop-e2e-pr.yml

```
@@ -126,7 +126,7 @@ jobs:
           containers: "trezor-user-env-unix bitcoin-regtest"
           workspace-dependencies: "@trezor/suite-desktop"
           test-group: ${{ matrix.TEST_GROUP }}
-          fail-fast: 3
+          fail-fast: 7
```

We have 7 agents. So 7 tests are started anyway.
So a completely broken build will run 7 tests anyway and then cancel rest.
Playwright does not cancel already running test with fail fast strategy
This will probably start few more tests, but i think the impact will be minimal.

My motivation is that I have added 4 tests verifying that the firmware version is ready. I am convinced it is preferable that those 4 will no fail fast everything. And that limit 3 was way to harsh.